workspace.go

```diff
@@ -513,6 +517,9 @@ type WorkspaceUpdateOptions struct {
 	// Optional: The time after which an automatic destroy run will be queued
 	AutoDestroyAt jsonapi.NullableAttr[time.Time] `jsonapi:"attr,auto-destroy-at,iso8601,omitempty"`
 
+	// Optional: The period of time to wait after workspace activity to trigger a destroy run.
```

Comment:
Can we describe the format of this field? Is it a go-style duration string using the time unit suffix?